### ISSUE TITLE
[Package request] its-pointless's packages

### ISSUE THREAD

**Issue by truboxl** [termux-user-repository/tur#42]
- [x] I understand [TUR package policy](https://github.com/termux-user-repository/tur/wiki/TUR-package-policy)

**Additional Info:**
Look into porting the remaining packages from https://github.com/its-pointless/its-pointless.github.io/blob/master/files/24/dists/termux/extras/binary-aarch64/Packages

**Comment by licy183** [comment]
Yeah, that's what I'm working on now. Some of these packages are built on device, but there is some problem if we use `termux-docker` to perform on-device building. I will write up about this later, maybe this night (GMT+8).

**Comment by licy183** [comment]
Following is a list of the packages from its-pointless's repo.

- arpack-ng: aarch64, arm, i686, x86_64
- arpack-ng-static: aarch64, arm, i686, x86_64
- asymptote: aarch64, arm, i686, x86_64
- bcal: aarch64, x86_64
- bthread: aarch64, arm, i686, x86_64
- bthread-static: aarch64, arm, i686, x86_64
- cksfv: aarch64, arm, i686, x86_64
- efte: aarch64, arm, i686, x86_64
- fp-compiler: aarch64, arm, i686, x86_64
- gcc-10: aarch64, arm, i686, x86_64
- gcc-11: aarch64, arm, i686, x86_64
- gcc-9: aarch64, arm, i686, x86_64
- glpk: aarch64, arm, i686, x86_64
- glpk-static: aarch64, arm, i686, x86_64
- gnat-10: aarch64, arm, i686, x86_64
- gnat-11: aarch64, arm, i686, x86_64
- gnat-9: aarch64, arm, i686, x86_64
- gnustep-base: aarch64, arm, i686, x86_64
- gnustep-make: aarch64, arm, i686, x86_64
- gobjc-10: aarch64, arm, i686, x86_64
- gobjc-9: aarch64, arm, i686, x86_64
- gprbuild: aarch64, arm, i686, x86_64
- groff: aarch64, arm, i686, x86_64
- julia: aarch64, arm, i686, x86_64
- julia-llvm: aarch64, arm, i686, x86_64
- libatomic-ops-dev: aarch64, arm, i686, x86_64
- libcgal: aarch64, arm, i686, x86_64
- libcgal-dev: aarch64, arm, i686, x86_64
- libcompiler-rt-termux: aarch64, arm, i686, x86_64
- libdvbpsi: aarch64, arm, i686, x86_64
- libdvbpsi-static: aarch64, arm, i686, x86_64
- libgccjit-10-dev: aarch64, arm, i686, x86_64
- libgccjit-11-dev: aarch64, arm, i686, x86_64
- libgccjit-9-dev: aarch64, arm, i686, x86_64
- libgfortran5: aarch64, arm, i686, x86_64
- libgfortran5-10: aarch64, arm, i686, x86_64
- libgfortran5-9: aarch64, arm, i686, x86_64
- libgnat-10: aarch64, arm, i686, x86_64
- libgnat-11: aarch64, arm, i686, x86_64
- libgnat-9: aarch64, arm, i686, x86_64
- libgomp-10: aarch64, arm, i686, x86_64
- libgomp-11: aarch64, arm, i686, x86_64
- libgomp-7: aarch64, arm, i686, x86_64
- libgomp-9: aarch64, arm, i686, x86_64
- libiberty-dev: aarch64, arm, i686, x86_64
- libllvm-julia: aarch64, arm, i686, x86_64
- libmpeg2: aarch64, arm, i686, x86_64
- libmpeg2-static: aarch64, arm, i686, x86_64
- libobjc4: aarch64, arm, i686, x86_64
- libobjc4-10: aarch64, arm, i686, x86_64
- libobjc4-9: aarch64, arm, i686, x86_64
- libopenspecfun: aarch64, arm, i686, x86_64
- libopenspecfun-static: aarch64, arm, i686, x86_64
- libsmb2: aarch64, arm, i686, x86_64
- libsmb2-static: aarch64, arm, i686, x86_64
- libsundials: aarch64, arm, i686, x86_64
- libsundials-static: aarch64, arm, i686, x86_64
- libtommath: aarch64, arm, i686, x86_64
- libtommath-dev: aarch64, arm, i686, x86_64
- lightning: aarch64, arm, i686, x86_64
- lightning-static: aarch64, arm, i686, x86_64
- moarvm: aarch64, arm, i686, x86_64
- mongodb: aarch64, x86_64
- mpich: aarch64, arm, i686, x86_64
- mpich-static: aarch64, arm, i686, x86_64
- nbtscan: aarch64, arm, i686, x86_64
- numpy: aarch64, arm, i686, x86_64
- ocaml: aarch64, arm, x86_64
- octave: aarch64, arm, i686, x86_64
- octave-static: aarch64, arm, i686, x86_64
- opam: aarch64, arm, x86_64
- openblas: aarch64, arm, i686, x86_64
- openblas-sandybridge: i686, x86_64
- openblas-sandybridge-static: i686, x86_64
- openblas-static: aarch64, arm, i686, x86_64
- opencv: aarch64, arm, i686, x86_64
- pari-gp: aarch64, arm, i686, x86_64
- picolisp: arm, i686
- portaudio: aarch64, arm, i686, x86_64
- portaudio-static: aarch64, arm, i686, x86_64
- potrace: aarch64, arm, i686, x86_64
- povray: aarch64, arm, i686, x86_64
- python2-numpy: aarch64, arm, i686, x86_64
- python2-scipy: aarch64, arm, i686, x86_64
- qhull: aarch64, arm, i686, x86_64
- qrupdate: aarch64, arm, i686, x86_64
- qrupdate-static: aarch64, arm, i686, x86_64
- quantlib: aarch64, arm, i686, x86_64
- quantlib-static: aarch64, arm, i686, x86_64
- r-base: aarch64, arm, i686, x86_64
- r-cran-rcpp: aarch64, arm, i686, x86_64
- r-cran-rquantlib: aarch64, arm, i686, x86_64
- r-cran-stringi: aarch64, arm, i686, x86_64
- r-cran-tidyverse: aarch64, arm, i686, x86_64
- r-cran-zoo: aarch64, arm, i686, x86_64
- rakudo: aarch64, arm, i686, x86_64
- scipy: aarch64, arm, i686, x86_64
- suitesparse: aarch64, arm, i686, x86_64
- uchardet: aarch64, arm, i686, x86_64
- uchardet-dev: aarch64, arm, i686, x86_64
- uchardet-static: aarch64, arm, i686, x86_64
- vlc: aarch64, arm, i686, x86_64
- xmlada: aarch64, arm, i686, x86_64
- xmlada-static: aarch64, arm, i686, x86_64


Here is the start: https://github.com/termux-user-repository/tur/pull/46.

**Comment by IntinteDAO** [comment]
@licy183 

gnustep-base: aarch64, arm, i686, x86_64
gnustep-make: aarch64, arm, i686, x86_64
vlc: aarch64, arm, i686, x86_64
povray: aarch64, arm, i686, x86_64
potrace: aarch64, arm, i686, x86_64
portaudio: aarch64, arm, i686, x86_64
portaudio-static: aarch64, arm, i686, x86_64

It's on Termux (gnustep-base is libgnustep-base)

bcal: aarch64, x86_64
https://github.com/termux/termux-packages/pull/11754

Waiting for merge

**Comment by licy183** [comment]
> @licy183
> 
> gnustep-base: aarch64, arm, i686, x86_64 gnustep-make: aarch64, arm, i686, x86_64 vlc: aarch64, arm, i686, x86_64 povray: aarch64, arm, i686, x86_64 potrace: aarch64, arm, i686, x86_64 portaudio: aarch64, arm, i686, x86_64 portaudio-static: aarch64, arm, i686, x86_64
> 
> It's on Termux (gnustep-base is libgnustep-base)
> 
> bcal: aarch64, x86_64 [termux/termux-packages#11754](https://github.com/termux/termux-packages/pull/11754)
> 
> Waiting for merge

Thanks for the information. I'll update that comment.

**Comment by defencedog** [comment]
Plz do something about `scipy`  ...80% of my engineering python packages are based on this package.

**Comment by IntinteDAO** [comment]
@defencedog 
This isn't package you want?
https://github.com/termux-user-repository/tur/tree/master/tur/python-scipy

**Comment by j0hnd3v** [comment]
> @licy183
> 
> gnustep-base: aarch64, arm, i686, x86_64 gnustep-make: aarch64, arm, i686, x86_64 vlc: aarch64, arm, i686, x86_64 povray: aarch64, arm, i686, x86_64 potrace: aarch64, arm, i686, x86_64 portaudio: aarch64, arm, i686, x86_64 portaudio-static: aarch64, arm, i686, x86_64
> 
> It's on Termux (gnustep-base is libgnustep-base)
> 
> bcal: aarch64, x86_64 [termux/termux-packages#11754](https://github.com/termux/termux-packages/pull/11754)
> 
> Waiting for merge

asymptote too

**Comment by joshcangit** [comment]
I've used `nbtscan` package.
Would be good to have it ported over.

**Comment by joshcangit** [comment]
FYI, for `fp-compiler` it should depend on `binutils-bin` rather than its metapackage.

**Comment by gety9** [comment]
@joshcangit were you able to install xidel?

**Comment by joshcangit** [comment]
I'm using the downloaded binary from the website.
But if you need to build it into a Termux package, then `fp-compiler` would be required.

**Comment by j0hnd3v** [comment]
Wow! This list has come a long way since I last saw it. I appreciate the time and effort you guys put into making these packages available again. If I had to point out one major thing that’s missing, it would be r-base and r-cran-*. Many people study R in various data science courses these days, and having this available on mobile devices would be invaluable.
